boards/arm/nucleo_h743zi/doc/index.rst

```diff
@@ -0,0 +1,204 @@
+.. _nucleo_h743zi_board:
+
+ST Nucleo H743ZI
+################
+
+Overview
+********
+
+The STM32 Nucleo-144 boards offer combinations of performance and power that
+provide an affordable and flexible way for users to build prototypes and try
+out new concepts. For compatible boards, the SMPS significantly reduces power
```

Comment:
What is the SMPS?

Reply:
according to the docs, the chip allows external Vcore supply via an SMPS (switch mode power supply) instead of using the internal LDO, to improve efficiency.

Reply:
https://www.st.com/content/ccc/resource/technical/document/application_note/group0/06/61/1a/a5/de/a1/4c/5e/DM00350156/files/DM00350156.pdf/jcr:content/translations/en.DM00350156.pdf

Reply:
Right, I think this abbreviation should be explained when used for the first time.

Reply:
Added meaning of `SMPS`